go/gazelle/add_dep.sh

```diff
@@ -0,0 +1,10 @@
+# To update external required dependencies with gazelle, follow these steps:
+# 1. Add the desired packages to your go.mod file with go get
+# 2. Run this shell script from the base directory typedb-driver to add the dependencies in go/deps.bzl with
+# bazel go_repositories
+# 3. Add the import to your go file
+
+# go get PACKAGE NAME, eg
+go get github.com/cucumber/godog
```

Comment:
Let's make these `go get` calls commented and only as optional things. Reread your instructions on top, please, and try imagining that you're not in the context of `go` dependencies and want to update the dependencies faster and smoother. Is it enough?